src/docs/concepts-development/content-management/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Content management in Flex
+slug: content-management
+updated: 2022-08-02
+category: concepts-development
+ingress:
+  This article introduces the Flex content management system and how
+  static pages can be managed in Flex Console
+published: true
+---
+
+In 2022-09, Flex introduces a headless CMS-like feature to create static
```

Comment:
This article has a draft being edited in https://docs.google.com/document/d/1Th6Eu4lhaIyplODU5qgpSmqApf48ZssceyCCzWyMqYE/edit?usp=sharing – the goal for this article is that it should be accessible to both technical and non-technical audiences. We need to decide if there's another concepts-type article that needs to happen for the technical audience, or if this is enough.